erpnext/erpnext_integrations/connectors/shopify_connection.py

```diff
@@ -244,6 +244,15 @@ def update_taxes_with_shipping_lines(taxes, shipping_lines, shopify_settings):
 	"""Shipping lines represents the shipping details,
 		each such shipping detail consists of a list of tax_lines"""
 	for shipping_charge in shipping_lines:
+		if shipping_charge.get("price"):
+			taxes.append({
+				"charge_type": _("Actual"),
+				"account_head": get_tax_account_head(shipping_charge),
+				"description": shipping_charge["title"],
+				"tax_amount": shipping_charge["price"],
+				"cost_center": shopify_settings.cost_center
+			})
+
 		for tax in shipping_charge.get("tax_lines"):
```

Comment:
The `shipping_charge` price contains the cumulative value from all the tax lines. I don't think we should add both.

```suggestion
	for shipping_charge in shipping_lines:
		shipping_tax_lines = [shipping_charge] or shipping_charge.get('tax_lines')
		for tax in shipping_tax_lines:
```

Reply:
We have a customer who apply the tax on shipping, so in his case we have to consider both shipping price as well tax applied on it.